new_dawn_server/actions/api/tests/test_like.py

```diff
@@ -79,8 +79,8 @@ def setUp(self):
             "action_type": ActionType.LIKE.value,
             "entity_id": 1,
             "entity_type": EntityType.MAIN_IMAGE.value,
-            "user_account_from": "/api/v1/account/1/",
-            "user_account_to": "/api/v1/account/2/"
+            "user_account_from": "1",
```

Comment:
Can we pass in integer here? Would that also work?

Reply:
The reason I pass string was because this could be replaced by hashed id in the future.

Reply:
Ok I'll buy that